examples/com/amazonaws/examples/EncryptionContextOverridesWithDynamoDBMapper.java

```diff
@@ -48,107 +48,73 @@ public static void main(String[] args) throws GeneralSecurityException {
     }
 
     public static void encryptRecord(final String cmkArn,
-                                     final String newEncryptionContextTableName,
                                      AmazonDynamoDB ddb,
                                      AWSKMS kms) throws GeneralSecurityException {
-        // Sample object to be encrypted
-        ExampleItem record = new ExampleItem();
-        record.setPartitionAttribute("is this");
-        record.setSortAttribute(55);
-        record.setExample("my data");
+        final String dynamoTableName = "DuckTable";
+        final String dynamoBackupName = "DuckBackup";
+
+        DuckPojo duckPojo = new DuckPojo();
+        duckPojo.setDuckName("Quackley");
+        duckPojo.setSwimming(true);
 
-        // Set up our configuration and clients
         final DirectKmsMaterialProvider cmp = new DirectKmsMaterialProvider(kms, cmkArn);
         final DynamoDBEncryptor encryptor = DynamoDBEncryptor.getInstance(cmp);
 
-        Map<String, String> tableNameEncryptionContextOverrides = new HashMap<>();
-        tableNameEncryptionContextOverrides.put("ExampleTableForEncryptionContextOverrides", newEncryptionContextTableName);
-        tableNameEncryptionContextOverrides.put("AnotherExampleTableForEncryptionContextOverrides", "this table doesn't exist");
-
-        // Supply an operator to override the table name used in the encryption context
-        encryptor.setEncryptionContextOverrideOperator(
-                overrideEncryptionContextTableNameUsingMap(tableNameEncryptionContextOverrides)
-        );
+        encryptor.setEncryptionContextOverrideOperator(EncryptionContextOperators.overrideEncryptionContextTableName(
+                dynamoBackupName, dynamoTableName));
 
-        // Mapper Creation
-        // Please note the use of SaveBehavior.PUT (SaveBehavior.CLOBBER works as well).
-        // Omitting this can result in data-corruption.
-        DynamoDBMapperConfig mapperConfig = DynamoDBMapperConfig.builder()
-                .withSaveBehavior(DynamoDBMapperConfig.SaveBehavior.PUT).build();
         DynamoDBMapper mapper = new DynamoDBMapper(ddb, mapperConfig, new AttributeEncryptor(encryptor));
```

Comment:
You deleted the creation of `mapperConfig` but did not replace it with anything.

Reply:
It is replaced with a static instance. Yes, it is tested.

Reply:
Whoops; that missed my text search.